### PR TITLE
Tweet for first unconference day going live

### PR DESCRIPTION
**Pull request by PatricHolmvall**
Announcement during first day of the 2021 unconference.

As always, feel free to edit and improve.

### CODE REVIEW

File: tweets/unconference2021-first-day-live.tweet [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The #NordicRSE unconference on research software is LIVE!
+
+We are excited to have @DrMowinckels @KristofferC89 @quantshah @AnneFouilloux kick off the event. Then the floor is yours!
```
Comment (bast) [inline tweets/unconference2021-first-day-live.tweet:3]:
```suggestion
We are excited to have @KristofferC89 @quantshah and @AnneFouilloux kick off the event. Then the floor is yours!
```

**Comment by samumantha** [comment]
this might be a language thing, but for me is LIVE somehow sounds like we are streaming it somewhere. maybe 'is starting now' ?

File: tweets/unconference2021-first-day-live.tweet [hunk 2]
```diff
@@ -0,0 +1,5 @@
+The #NordicRSE unconference on research software is LIVE!
```
Comment (PatricHolmvall) [inline tweets/unconference2021-first-day-live.tweet:1]:
```suggestion
The #NordicRSE unconference on research software is starting now!
```

**Comment by PatricHolmvall** [comment]
> this might be a language thing and totally fine as is for everyone else, but for me is LIVE somehow sounds like we are streaming it somewhere. maybe 'is starting now' ?

Fixed!

**Pull request merged**